app/controllers/editor/categories_controller.rb

```diff
@@ -43,6 +43,24 @@ def destroy
     redirect_to editor_categories_url, notice: "カテゴリが削除されました"
   end
 
+  # move to left
+  def move_higher
```

Comment:
Do you mean that we should create `Editor::Categories::MoveHigherController#update` or something?